Sources/StringsLintFramework/Parsers/StringsJSONCommentParser.swift

```diff
@@ -57,6 +58,17 @@ public struct StringsJSONCommentParser: LocalizableParser {
         return strings
     }
 
+  private func buildMapLineNumberMap(from file: File) -> [String: Int] {
+    var map = [String: Int]()
+
+    file.lines.enumerated().forEach {
+      guard let localizedKey = $0.element.localizedKey else { return }
+      map.updateValue($0.offset+1, forKey: localizedKey)
+    }
+
+    return map
+  }
```

Comment:
@harkmall this reduces the runtime from 26s to 1 when we are only parsing one string file!

Reply:
dope!